ipywidgets/static/widgets/js/manager.js

```diff
@@ -295,8 +158,8 @@ define([
     /**
      * Deprecated, use `new_widget` instead.
      */
-    WidgetManager.prototype.create_model = function (options) {
-        console.warn('WidgetManager.create_model is deprecated. Use WidgetManager.new_widget');
+    ManagerBase.prototype.create_model = function (options) {
+        console.warn('ManagerBase.create_model is deprecated. Use ManagerBase.new_widget');
```

Comment:
Could we move this to WidgetManager, so that we don't carry deprecated classes to the cleaner future?